nodes/src/nodes/llm_openai_api/openai_client.py

```
@@ -0,0 +1,78 @@
+
+        # Save our chat class into the bag
+        bag['chat'] = self
+
+    def is_retryable_error(self, error):
+        """
+        Determine if the error is retryable.
+        """
+        if isinstance(error, AuthenticationError):
+            return False
+        elif isinstance(error, APIError):
+            return False
```

Ruff (0.15.7)

[warning] Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)